src/markdown-pages/terraform/terragrunt-configuration.mdx

```diff
@@ -255,24 +273,40 @@ Now, when you run `terragrunt plan`, Terragrunt will provide the input block's v
 [output] "terraform apply" is subsequently run.
 ```
 
-Run `terragrunt apply`. After Terraform finishes process, check your New Relic account for the new Demo Policy.
-</Step>
+Terragrunt provides the `inputs` block's values.
 
-<!-- ADDING TO YOUR CONFIGURATION -->
+</Step>
 
 <Step>
 
-## Adding to your configuration
+Run `terragrunt apply`:
+
+```bash
+terragrunt apply
+```
+
+Now your Demo Policy is in your New Relic account.
+
+</Step>
+
+</Steps>
+
+## Add to your configuration
 
-Now that you have a basic New Relic configuration created, add configurations built in the [Getting Started with Terraform](/terraform/get-started-terraform) and [Terraform modules](/terraform/terraform-modules) guides in this step.
+Now that you've created a basic New Relic configuration, add the configurations from our [Getting Started with Terraform](/terraform/get-started-terraform) and [Terraform modules](/terraform/terraform-modules) guides.
 
 <Callout variant="tip">
 
-If you haven't done the previous developer guides, you can copy the configurations for this section from the [Terragrunt Intro Github Repo](https://github.com/jsbnr/nr-terragrunt-intro-example).
+If you haven't done these guides yet, you can copy their configurations from the [Terragrunt intro Github repo](https://github.com/jsbnr/nr-terragrunt-intro-example).
 
 </Callout>
 
-Update the __main.tf__ to include the additional terraform configurations. Update the email address in the alert channel to your preferred email address:
+<Steps>
+
+<Step>
+
+In _main.tf_, update the email address in the alert channel to your preferred email address:
```

Comment:
_main.tf_ is inside src folder. I suggested adding this reference for clarification.